protos/v1alpha1/resource_mapping.proto

```diff
@@ -49,6 +49,18 @@ message Resource {
   // Required. The full resource name. Example:
   // `//spanner.googleapis.com/projects/p1/instances/i1/databases/d1`
   string name = 2;
+
+  // Optional. The representation for subresource in the format of a URL. 
```

Comment:
Could you format the comment to limit to 80 characters? Also add a few empty lines in between examples to make it more readable.